### PR TITLE
[TypeScript Migration][AE-239] Loading Component

### PR DESCRIPTION
**Pull request by beau-charman**
JIRA: https://aesoponline.atlassian.net/browse/AE-239

### CODE REVIEW

File: src/components/Loading/Loading.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React, { forwardRef } from 'react';
+import cx from 'classnames';
+import { useThemeContext } from '~/contexts';
+import type { LoadingProps } from './Loading.types';
+import styles from './Loading.module.css';
+
+const Loading = forwardRef<HTMLDivElement, LoadingProps>(function LoadingRef({
```
Comment (MkMan) [inline src/components/Loading/Loading.tsx:7]: Should the ref type be `HTMLSpanElement` instead?

File: src/components/Loading/Loading.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  className,
+  isLoading,
+  shouldFillSpace = false,
+  size = 'medium',
+  theme,
+}) {
```
Comment (MkMan) [inline src/components/Loading/Loading.tsx:13]: Don't forget to capture and use the `ref` argument 🙂

File: src/components/Loading/Loading.tsx (hunk 1 above)
Comment (beau-charman) [reply src/components/Loading/Loading.tsx:7]: Thank you for the pick up. I might just remove the `forwardRef`.

File: src/components/Loading/Loading.tsx (hunk 2 above)
Comment (beau-charman) [reply src/components/Loading/Loading.tsx:13]: yeah, as above I should havent even have included it

**Review by MkMan — APPROVED** [review]

**Pull request merged**